packages/react-dev-utils/launchEditor.js

```diff
@@ -11,10 +11,62 @@
 const fs = require('fs');
 const path = require('path');
 const child_process = require('child_process');
+const EventEmitter = require('events').EventEmitter;
 const os = require('os');
 const chalk = require('chalk');
 const shellQuote = require('shell-quote');
 
+// Inspired by https://github.com/rannn505/node-powershell
```

Comment:
Any reason we're not just using it?

Reply:
In that module you have an method `addCommand` and an method `invoke`.
With `addCommand` you can add a few commands and execute them with `invoke` at once getting the output of all commands. I wanted a clear connection between a command and its result.

Reply:
Maybe we could send a pull request to the module, adding a `execute` shortcut which calls `addCommand` and `invoke`?